package.json

```diff
@@ -28,6 +27,7 @@
     "react-dom": "^16.13.0",
     "sanitize-html": "^2.1.2",
     "seamless-immutable": "^7.1.4",
+    "string_decoder": "^1.3.0",
```

Comment:
:thinking: is this one expected?

Reply:
Yes, the code would not compile if I didn't include this package after I uninstalled react-styled-system, I guess it's some dependency spaghetti.

Reply:
oh I see. a breaking change from webpack 5